src/elements/EditableTable.tsx

```diff
@@ -0,0 +1,262 @@
+import Box, { BoxProps } from "@vertigis/web/ui/Box";
+import { FormElementProps, FormElementRegistration } from "@geocortex/workflow/runtime";
+import Input from "@vertigis/web/ui/Input";
+import React, { useState, useEffect } from "react";
+import Table, { TableProps } from "@vertigis/web/ui/Table";
+import TableBody from "@vertigis/web/ui/TableBody";
+import TableCell, { TableCellProps } from "@vertigis/web/ui/TableCell";
+import TableFooter from "@vertigis/web/ui/TableFooter";
+import TableHead from "@vertigis/web/ui/TableHead";
+import TableRow from "@vertigis/web/ui/TableRow";
+
+type FooterRow = Record<string, any>;
+type Row = Record<string, any>;
+type SettableBoxProps = Pick<BoxProps, "maxHeight" | "maxWidth">;
+type SettableTableProps = Pick<TableProps, "size" | "stickyHeader">;
+
+interface Column {
+    alias?: string;
+    align?: TableCellProps["align"];
+    format?: (value: any) => any;
+    parse?: (
+        event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+        row: Row,
+        column: Column
+    ) => any;
+    props?: any;
+    columnCalculation?: "sum" | "average" | ((values: any) => any);
+    editable: boolean;
```

Comment:
`editable` and `type` should be optional.

Reply:
done.